tests/infra/consortium.py

```diff
@@ -528,11 +528,14 @@ def set_js_app_from_json(
         # Large apps take a long time to process - wait longer than normal for commit
         return self.vote_using_majority(remote_node, proposal, careful_vote, timeout=30)
 
-    def set_js_runtime_options(self, remote_node, max_heap_bytes, max_stack_bytes):
+    def set_js_runtime_options(
+        self, remote_node, max_heap_bytes, max_stack_bytes, max_execution_time
+    ):
         proposal_body, careful_vote = self.make_proposal(
             "set_js_runtime_options",
             max_heap_bytes=max_heap_bytes,
             max_stack_bytes=max_stack_bytes,
+            max_execution_time_ms=max_execution_time,
```

Comment:
I think we should call the Python parameter `max_execution_time_ms` as well. It's confusing to have a subtle rename for just this field here, and elsewhere Python time args are usually in seconds.

Reply:
Done